components/booking/DatePicker.tsx

```diff
@@ -0,0 +1,134 @@
+import { ChevronLeftIcon, ChevronRightIcon } from "@heroicons/react/solid";
+import { useEffect, useState } from "react";
+import dayjs, { Dayjs } from "dayjs";
+import utc from "dayjs/plugin/utc";
+import timezone from "dayjs/plugin/timezone";
+import getSlots from "@lib/slots";
+dayjs.extend(utc);
+dayjs.extend(timezone);
+
+const DatePicker = ({
+  weekStart,
+  onDatePicked,
+  workingHours,
+  organizerTimeZone,
+  inviteeTimeZone,
+  eventLength,
+}) => {
+  const [calendar, setCalendar] = useState([]);
+  const [selectedMonth, setSelectedMonth]: number = useState();
+  const [selectedDate, setSelectedDate]: Dayjs = useState();
+
+  useEffect(() => {
+    setSelectedMonth(dayjs().tz(inviteeTimeZone).month());
+  }, []);
```

Comment:
Codacy has a fix for the issue: [React Hook useEffect has a missing dependency: 'inviteeTimeZone'. Either include it or remove the dependency array. If 'setSelectedMonth' needs the current value of 'inviteeTimeZone', you can also switch to useReducer instead of useState and read 'inviteeTimeZone' in the reducer.](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689306)
```suggestion
  }, [inviteeTimeZone]);
```